CMakeLists.txt

```diff
@@ -72,12 +72,22 @@ option(OCR           "enable OCR, requires OMR" OFF)           # requires tesser
 option(SOUNDFONT3    "ogg vorbis compressed fonts" ON)         # enable ogg vorbis compressed fonts, require ogg & vorbis
 option(HAS_AUDIOFILE "enable audio export" ON)                 # requires libsndfile
 option(USE_SYSTEM_QTSINGLEAPPLICATION "Use system QtSingleApplication" OFF)
-option(BUILD_LAME    "enable mp3 export" ON)                   # requires libmp3lame
 
 SET(JACK_LONGNAME "jack (jack audio connection kit)")
 SET(JACK_MIN_VERSION "0.98.0")
 option(BUILD_JACK    "Build with support for ${JACK_LONGNAME}. jack >= ${JACK_MIN_VERSION} will be needed." ON)
 
+# NON-FREE COMPONENTS. (Option to disable is required for packaging on some platforms)
+if (${CMAKE_NON_FREE_LIBS} MATCHES "DISABLED")
```

Comment:
Why introduce a new environment variable CMAKE_NON_FREE_LIBS and not use the existing option BUILD_LAME like this `-DBUILD_LAME=OFF` ?